Source/Core/Core/PowerPC/JitArm64/JitArm64_FloatingPoint.cpp

```diff
@@ -403,12 +403,12 @@ void JitArm64::ConvertDoubleToSingleLower(ARM64Reg dest_reg, ARM64Reg src_reg)
 {
   FlushCarry();
 
-  const BitSet32 gpr_saved = gpr.GetCallerSavedUsed();
+  const BitSet32 gpr_saved = gpr.GetCallerSavedUsed() & BitSet32(0x4000000F);
```

Comment:
Which registers does 0x4000000F represent? Maybe add a comment or give it a name.

Reply:
X0-X3 (as documented in a comment above `JitArm64::GenerateConvertDoubleToSingle`) and X30 (LR).

If I were to give it a name, what would that name look like? Mentioning all the registers in the name feels a bit awkward...

Reply:
You could duplicate the comment.

Reply:
I like that idea.